### PR TITLE
Show market price and notional in trade widget

### PR DESCRIPTION
**Pull request by antonyshchenko**
Screenshot 2024-04-24 at 21 33 16

Screenshot 2024-04-24 at 21 33 09

Screenshot 2024-04-24 at 21 33 41

### CODE REVIEW

**Review by imykolenko — APPROVED** [review]
looks great!

File: web-ui/src/components/Screens/HomeScreen/TradeWidget.tsx [hunk 1]
```diff
@@ -225,15 +286,50 @@ export default function TradeWidget({
...
-          <p className="text-center text-white">
-            {`${side == 'Buy' ? 'Buying' : 'Selling'} ${amount} ${
-              baseSymbol.name
-            } ${
-              isMarketOrder
-                ? '(market order) '
-                : `for ${price} ${quoteSymbol.name}`
-            }`}
-          </p>
+          <div className="text-white">
```
Comment (bholzman) [inline web-ui/src/components/Screens/HomeScreen/TradeWidget.tsx:289]: suggest adding `text-center` here

**Review by bholzman — APPROVED** [review]
Looks great!

File: web-ui/src/components/Screens/HomeScreen/TradeWidget.tsx (hunk 1 above)
Comment (antonyshchenko) [reply web-ui/src/components/Screens/HomeScreen/TradeWidget.tsx]: Done, also made the text size smaller

**Pull request merged**